diff --git a/node_modules/pdf-parse/index.js b/node_modules/pdf-parse/index.js
index e9fc367..ea36063 100644
--- a/node_modules/pdf-parse/index.js
+++ b/node_modules/pdf-parse/index.js
@@ -9,7 +9,7 @@ let isDebugMode = !module.parent;
 
 
 //for testing purpose
-if (isDebugMode) {
+if (false && isDebugMode) {
 
     let PDF_FILE = './test/data/05-versions-space.pdf';
     let dataBuffer = Fs.readFileSync(PDF_FILE);
